queue/python/queue.py

```diff
@@ -0,0 +1,40 @@
+# Implementation of queue using Linked List:
+class Node:
+    def __init__(self, data):
+       self.data = data
+       self.next = None
+class Queue:
+    def __init__(self):
+        self.head = None
+        self.last = None
+    def enqueue(self, data):
+        if self.last is None:
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpvcelzp9o/queue/python/queue.py
+++ b/tmp/tmpvcelzp9o/queue/python/queue.py
@@ -7,6 +7,7 @@
     def __init__(self):
         self.head = None
         self.last = None
+
     def enqueue(self, data):
         if self.last is None:
             self.head = Node(data)
```